frontend/webapp/containers/main/actions/choose-action-body/custom-fields/pii-masking.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { InputList, Text } from '@/reuseable-components';
+
+const FieldWrapper = styled.div`
```

Comment:
the style is common, you can move it to a new file and use it in all the relevant files